gap/digraph.gi

```diff
@@ -429,6 +444,98 @@ DigraphCons);
 InstallMethod(Digraph, "for a list, list, and list", [IsList, IsList, IsList],
 {dom, src, ran} -> DigraphCons(IsImmutableDigraph, dom, src, ran));
 
+InstallMethod(Digraph, "for a string naming a digraph", [IsString],
+function(name)
+  # edge case to avoid interfering with Digraph([])
+  if name = "" then
+    TryNextMethod();
+  fi;
+
+  # standardise string format to search database
+  name := LowercaseString(name);
+  RemoveCharacters(name, " \n\t\r");
+
+  # load database if not already done
+  DIGRAPHS_LoadNamedDiSparse6Strings();
+
+  if not name in RecNames(DIGRAPHS_NamedDiSparse6Strings) then
+    ErrorNoReturn("Named digraph not found. Please check argument 'name',\n",
+                  "or view list of available digraphs with names containing\n",
+                  "string s using ListNamedDigraphs(s).");
+  fi;
+  return DigraphFromDiSparse6String(DIGRAPHS_NamedDiSparse6Strings.(name));
+end);
+
+InstallMethod(ListNamedDigraphs,
+"for a partially completed string and a pos int",
```

Comment:
These messages (which are not very important, to be honest) are really meant to be there just to describe the type of GAP object in the follow line, rather than give their purpose/mathematical meaning.
```suggestion
"for a string and a pos int",
```